Resources/doc/reference/command.rst

```diff
@@ -0,0 +1,28 @@
+Command
+=======
+
+You can create a dump file with products schema information. Even those created with the previous command:
```

Comment:
But I don't get why products? Does this concern only products? And what about the thing with the previous command?